orm/connection.go

```diff
@@ -0,0 +1,71 @@
+package orm
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+
+	_ "github.com/joho/godotenv/autoload"
+	"gorm.io/driver/mysql"
+	"gorm.io/gorm"
+)
+
+var connection *gorm.DB
+var databaseName = "notifications"
+var Charset = "utf8mb4"
+var Collation = "utf8mb4_bin"
```

Comment:
Also I wonder if we can add the database name, character set, and collation as environment variables to the `.env.template` file since just as the `BEB_ENV` environment variable they all seem to be used in multiple locations such as in this file and in the `setup.sql.sh` file.  This way we do not have to constantly hard code these values.

Reply:
Hmmm I'm a little wary of letting the user/admin decide these things. It will make it hard to replicate and track down bugs if everyone is using different settings. And as for the name, I'm not sure I see what value the user gets out of it. If they have a name collision, they can change the BEB_ENV variable. I'm not totally against the idea but I'm not sure I see the pros vs. the cons. The existence of the variables means no one has to hard code anything, outside of these few lines.

If we do get admins who really want this flexibility, I don't think it will be hard to patch that capability later.

Reply:
Oh I never thought about it that way of it letting the user/admin decide the values for these variables.  I just thought instead of having these values in multiple locations to just have them in one location so that if you need to update them then there is only one place to go to update.  From where I come I am accustomed to having some of these values set as environment variables especially the database name so that if a user/admin wants to run the application then they would know what they would need to setup such as for the database name if the database is not automatically created by the application a user/admin would then know what name to give the database when they create it.  I guess in this case we cannot be certain which or what user/admin will be running this application and how they will be running it so we might want to be more protective of some of these values.